llvm/lib/SYCLLowerIR/LowerWGLocalMemory.cpp

```diff
@@ -52,6 +54,32 @@ ModulePass *llvm::createSYCLLowerWGLocalMemoryLegacyPass() {
   return new SYCLLowerWGLocalMemoryLegacy();
 }
 
+static bool inlineAllocateLocalMemoryFunc(Module &M) {
```

Comment:
Please add comment to describe why we need to inline the func first.

Reply:
done